package.json

```diff
@@ -48,7 +48,7 @@
     "hoek": "^5.0.4",
     "joi": "^13.7.0",
     "request": "^2.88.0",
-    "screwdriver-data-schema": "^18.38.0",
+    "screwdriver-data-schema": "^19.12.1",
```

Comment:
do we wanna bump this up now ? or wait for these scm fixes to go through ?

Reply:
later
the newest one is v20